### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.2.2.2 - provide stop/pause for carousel

### ISSUE THREAD

**Issue by SarahEdding** [benjaminkott/bootstrap_package#1285]
# Feature Request

## Description

All moving content must provide an option for the user to stop and/or pause the movement. The autoplay caraousel currently does not offer this. 

## Solution

Recommendation: disable autoplay and do not offer it to editors. If autoplay must be possible, provide an easiliy perceivable, keyboard operable pause button in direct proximity with the carousel (Frontend). https://www.w3.org/WAI/WCAG21/Techniques/general/G4

## Note
The carousel has a number of other issues, such as poor colour contrast, a flaw in html semantics, a missing aria-role and no stop/pause button. Recommendation: rebuild carousel according to W3C tutorial https://www.w3.org/WAI/tutorials/carousels/)

**Comment by SvenJuergens** [comment]
hi @SarahEdding maybe a question and an alternative solution suggestion.
"All moving content must provide an option for the user to stop and/or pause the movement." But what if the content, in this case the carousel, does not move automatically?
Bootstrap itself offers a solution that could be used. https://getbootstrap.com/docs/5.3/getting-started/accessibility/#reduced-motion
The browser support for this is very good and so the question is whether this is a way to meet the accessibility requirements?

**Comment by SarahEdding** [comment]
Hi @SvenJuergens 
I would definitely recommend for the carousel to only rotate on click/swipe. In that case, no extra pause button need be provided. 

The trouble with the prefers-reduced-motion media query is, it only displays the non-rotating version to users who have reduced motion settings active in their operating system. 
We need a solution where any user can stop the motion at any point in time, for instance if it takes them a little longer to read the slide. Or if they can't focus on reading while the motion in their peripheral vision distracts them. 

So: If we can deactivate auto-rotation completely, we'd be compliant with this criterion. 
If this only works for users with prefers-reduced-motion, we still have to provide an extra pause button.

**Comment by SvenJuergens** [comment]
Hi @SarahEdding 
That's right, it would only help those who have the "prefers-reduced-motion" enabled in the operating system.  
With a pause button you would reach more people.

**Comment by benjaminkott** [comment]
Autoplay is now disabled by default, a new option has been added to the backend to enable it on demand. It states that this is not recommended.

**Comment by SarahEdding** [comment]
QA done, all fixed
